### ISSUE TITLE
Support for micro transactions?

### ISSUE THREAD

**Issue by goldfire** [greenheartgames/greenworks#141]
Is it possible to listen for the `MicroTxnAuthorizationResponse_t` event without a custom build of Greenworks? As suggested in #59, we are using the following to try and listen for a successful transaction:

```javascript
greenworks.on('micro-txn-authorization-response', (orderId, authorized) => {
  // ...
});
```

However, this never gets fired, which I suppose is because it isn't explicitly setup in Greenworks. I couldn't tell from the other issue if @bgSosh ever got this figured out, but I've seen other Greenworks games using micro transactions with the overlay.

**Comment by hokein** [comment]
@goldfire  `micro-txn-authorization-response` event is not implemented in upstream Greenworks. So the event will never get emitted. I think it makes sense to implement it.

@bgSosh Would you like to contribute your implementation to upstream as you seem have implemented it https://github.com/greenheartgames/greenworks/issues/59#issuecomment-155819565? Thanks.

**Comment by MadSpyxFR** [comment]
@hokein  Please, check #142 this PR is an adaptation from my steam integration

**Comment by hokein** [comment]
Fixed in #142.
